weave-js/src/components/PagePanelComponents/Home/Browse2/RunsTable.tsx

```diff
@@ -153,14 +153,11 @@ export const RunsTable: FC<{
       {
         field: 'status_code',
         headerName: '',
-        width: 40,
-        minWidth: 40,
-        maxWidth: 40,
         sortable: false,
         disableColumnMenu: true,
         resizable: false,
         renderCell: cellParams => {
-          return <CallStatusCodeChip statusCode={cellParams.row.status_code} />;
+          return <StatusChip value={cellParams.row.status_code} />;
```

Comment:
personally, i kind of like `iconOnly` here to try to minimize the left-side space as much as possible. Not a blocking comment.

Reply:
I'd probably lean towards that too but was trying to move towards alignment with what the design team has in https://www.figma.com/proto/27JVuBYWHbcAYc8WPb4HqA/Weaveflow-hifi?page-id=1024%3A116040&type=design&node-id=1029-126886&viewport=714%2C814%2C0.17&t=UWlDYqfUwYyq39jU-9&scaling=min-zoom&starting-point-node-id=1029%3A126886&show-proto-sidebar=1

Let's add it to agenda for when we meet with them.